proposals/primary-constructors.md

```diff
@@ -71,13 +71,34 @@ struct_body
   : '{' struct_member_declaration* '}' ';'?
   | ';'
   ;
+  
+interface_declaration
+  : attributes? interface_modifier* 'partial'? 'interface'
+    identifier variant_type_parameter_list? interface_base?
+    type_parameter_constraints_clause* interface_body
+  ;  
+    
+interface_body
+  : '{' interface_member_declaration* '}' ';'?
+  | ';'
+  ;
```

Comment:
```suggestion
interface_declaration
  : attributes? interface_modifier* 'partial'? 'interface'
    identifier variant_type_parameter_list? interface_base?
    type_parameter_constraints_clause* interface_body? ';'?
  ;  
    
interface_body
  : '{' interface_member_declaration* '}'
  ;
```

Potential other representation.  Yours is better in that it always specifies taht one of the two is needed.   I like mine though for the simplicity around the semicolon.  However, mine would need normative text saying at least one of the body/semicolon must be provided.

Reply:
I am following the pattern used for class/struct.